pgvectorscale/src/access_method/upgrade_test.rs

```diff
@@ -22,9 +22,8 @@ pub mod tests {
     }
 
     #[test]
-    #[ignore]
     ///This function is only a mock to bring up the test framewokr in test_delete_vacuum
-    fn test_upgrade() {
+    fn test_upgrade_from_0_0_2() {
```

Comment:
The logic here is very similar to the new `test_upgrade_base` - would it be possible to merge those into one function that checks the version and only does the extra work for 0.0.2?